### ISSUE TITLE
Is it possible to create an interger only uuid?

### ISSUE THREAD

**Issue by Wizzel1** [Daegalus/dart-uuid#68]

**Comment by Daegalus** [comment]
I don't think so. A uuid is just the Hex encoded representation of a bunch of random bytes. It's hard to near impossible to consistently generate them as integers.

Also any other form is not valid uuid.

You can try to use the byte list form internally for things. 

You could also break standard and try to encode the data into Long bits. But that probably won't work.

I don't think uuid is the right choice if you can't have letters.

**Comment by Wizzel1** [comment]
Got it, thanks.
